docs/standard/linq/project-anonymous-type.md

```diff
@@ -0,0 +1,76 @@
+---
+title: How to project an anonymous type - LINQ to XML
+description: You can project to an anonymous type rather than create a type just for use in the projection. This article provides an example for C# and Visual Basic.
+ms.date: 07/20/2015
+dev_langs:
+  - "csharp"
+  - "vb"
+ms.assetid: 5cb9be13-5ac4-4373-a034-b3520a5b2dec
+---
+# How to project an anonymous type (LINQ to XML)
+
+In some cases you might want to project a query to a new type, but the query would be your only use for the new type. Rather than create the type, you can project to an anonymous type. Anonymous types provide a convenient way to encapsulate a set of read-only properties in an object without having to explicitly define a type first. If you write a query that creates an object of an anonymous type in the `select` clause, the query returns an `IEnumerable` of the type.
```

Comment:
```suggestion
In some cases, you might want to project a query to a new type, but the query would be your only use for the new type. Rather than create the type, you can project to an anonymous type. Anonymous types provide a convenient way to encapsulate a set of read-only properties in an object without having to explicitly define a type first. If you write a query that creates an object of an anonymous type in the `select` clause, the query returns an <xref:System.Collections.IEnumerable> of the type.
```